### PR TITLE
typo: iothub/resource-manager/Microsoft.Devices

### PR DESCRIPTION
**Pull request by nschonni**
- Double word "the"
- Thehub -> The hub
- Double word "state"

### CODE REVIEW

**Comment by nschonni** [comment]
Don't get the Go failure
```
INFO:swaggertosdk.autorest_tools:FATAL: System.InvalidOperationException: codegen for preview swagger Microsoft.Devices/preview/2018-12-01-preview/iothub.json must be under a preview subdirectory
```
It's saying it's not under a preview folder, but it is

File: specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json [hunk 1]
```diff
@@ -2386,7 +2386,7 @@
           "minimum": 10485760
         },
         "encoding": {
-          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro' and 'avrodeflate'. Default value is 'avro'.",
+          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro' and 'avroDeflate'. Default value is 'avro'.",
```
Comment (amarzavery) [inline specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json]: I am not sure if the service supports avro**D**eflate. Please revert this change.

Comment (amarzavery) [inline specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json]: not sure if avroDeflate is valid. Please keep it as-is.

**Review by amarzavery — APPROVED** [review]
